ui/imports/shared/popups/send/views/FeesView.qml

```diff
@@ -7,80 +7,95 @@ import StatusQ.Core.Theme 0.1
 
 import utils 1.0
 
-import shared.stores 1.0
-
-import "../controls"
+import shared.popups.send.controls 1.0
 
 Rectangle {
     id: root
 
     property double gasFiatAmount
     property bool isLoading: false
     property var bestRoutes
-    property var store
-    property var currencyStore: store.currencyStore
     property var selectedAsset
     property int errorType: Constants.NoError
+    property string currentCurrency
```

Comment:
This property is probably not needed as the only need is to pass it back to the callback.